docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,61 @@
+[//]: # (title: Troubleshooting)
+
+Most of the browsers support WebAssembly. For recent browser versions, WebAssembly works by default. However,
+for older browser versions, you still need to update the browser settings.
+
+## Browser versions
+
+To run a Kotlin/Wasm project, check if WebAssembly works with the browser version by default. Otherwise, update the 
+settings of the target environment:
+
+<tabs>
+<tab title="Chrome">
+
+* For version 119 or later:
+
+  WebAssembly works by default.
```

Comment:
It's not about WebAssembly but Kotlin/Wasm binaries or supported and enabled by default proposals

Reply:
Changed it.